pkg/api/server.go

```diff
@@ -155,23 +167,14 @@ func (s *Server) handleFaultsRequest(w http.ResponseWriter, r *http.Request) {
 	startEpoch, endEpoch := computeSpanFromRequest(startEpochRequest, endEpochRequest, epochSpanRequest, currentEpoch)
 	faults := s.analyzer.GetFaults(startEpoch, endEpoch)
 
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusOK)
-
 	response := FaultsResponse{
 		Span: Span{
 			Start: startEpoch,
 			End:   endEpoch,
 		},
 		FaultRecord: faults,
 	}
-	encoder := json.NewEncoder(w)
-	encoder.SetIndent("", "  ")
```

Comment:
this was added explicitly so the output is more readable -- it may not stay forever but useful for now... can we add back into `respondOK`?

Reply:
added back